### PR TITLE
Remove hello_leds example

### PR DESCRIPTION
**Pull request by mnishiguchi**
Related PR: https://github.com/nerves-project/nerves_examples/pull/251

### CODE REVIEW

**Comment by fhunleth** [comment]
This should also need the CI steps removed at https://github.com/nerves-project/nerves_examples/blob/main/.circleci/config.yml#L147-L174. There might be other places if you grep for `hello_leds`.

File: .circleci/config.yml [hunk 1]
```diff
@@ -144,9 +144,6 @@ collect_stats: &collect_stats
     - restore_cache:
         key: nerves/stats/{{ checksum ".project" }}/{{ checksum ".target" }}-{{ .Revision }}-{{ .Environment.CIRCLE_TAG }}
 
-    - run:
-        name: hello_leds
-        command: echo "hello_leds" > .project
     - run: echo "rpi0" > .target
     - restore_cache:
         key: nerves/stats/{{ checksum ".project" }}/{{ checksum ".target" }}-{{ .Revision }}-{{ .Environment.CIRCLE_TAG }}
```
Comment (fhunleth) [inline .circleci/config.yml:149]: These lines and down to the blank line below need to be removed too.

Comment (mnishiguchi) [reply .circleci/config.yml]: oops!

**Comment by fhunleth** [comment]
Looks good!

**Pull request merged**